sklearn/base.py

```diff
@@ -578,3 +578,21 @@ def is_regressor(estimator):
         True if estimator is a regressor and False otherwise.
     """
     return getattr(estimator, "_estimator_type", None) == "regressor"
+
+
+def is_pairwise(estimator):
+    """Returns True if the given estimator has a _pairwise attribute
```

Comment:
Pep257: summary should be one line only. More description can follow after a blank line